### PR TITLE
BEN-2516 - make tree rebuild more verbose.  ignore deleted folders, same as django-mptt's rebuild

### PR DESCRIPTION
**Pull request by mnadasan**
-  use partial_rebuild which is now available

### CODE REVIEW

File: filer/utils/checktrees.py [hunk 1]
```diff
@@ -53,12 +54,13 @@ def get_corrupted_root_nodes(self):
             pk__in=list(self.corrupted_folders.keys())).\
             values_list('tree_id', flat=True).distinct()
         return self.manager.filter(
-            parent__isnull=True, tree_id__in=corrupted_trees)
+            parent__isnull=True, deleted_ta__isnull=True, tree_id__in=corrupted_trees)
```
Comment (bogdan-cornianu) [inline filer/utils/checktrees.py]: I think there's a typo in `deleted_ta__isnull`; should be `deleted_at__isnull`.

File: filer/utils/checktrees.py [hunk 2]
```diff
@@ -1,4 +1,5 @@
 from django.db.models import Count
+from filer.models import Folder
```
Comment (bogdan-cornianu) [inline filer/utils/checktrees.py]: Where do we use this import?

Comment (bogdan-cornianu) [reply filer/utils/checktrees.py]: If we do need this import, I think the correct package is `from filer.models.foldermodels import Folder`

Comment (mnadasan) [reply filer/utils/checktrees.py]: Not used. I'll remove it.

**Review by bogdan-cornianu — APPROVED** [review]

**Review by stefaniloaie — APPROVED** [review]

**Pull request merged**